src/common/common.h

```diff
@@ -12,14 +12,16 @@ std::optional<RECT> get_button_pos(HWND hwnd);
 std::optional<RECT> get_window_pos(HWND hwnd);
 // Gets mouse postion.
 std::optional<POINT> get_mouse_pos();
-// Gets active window, filtering out all "non standard" windows like the taskbar, etc.
-HWND get_filtered_active_window();
+
 // Gets window ancestor (usualy the window we want to do stuff with), filtering out all "non standard" windows like the taskbar, etc. and provide the app process path
 struct WindowAndProcPath {
   HWND hwnd = nullptr;
   std::wstring process_path;
 };
 WindowAndProcPath get_filtered_base_window_and_path(HWND window);
+// Gets active window, filtering out all "non standard" windows like the taskbar, etc.
```

Comment:
Can you expand the comment, it's not intuitive to figure out what exactly this method does and what the return value is.

Reply:
This would not be a change related to this PR, I'll make a separate one for this and some other SCG tweak.

Reply:
OK